content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -1317,6 +1330,49 @@ created Pod should be ready without any of its containers crashing, for it to be
 This defaults to 0 (the Pod will be considered available as soon as it is ready). To learn more about when
 a Pod is considered ready, see [Container Probes](/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
 
+### Pod Replacement Policy
+
+{{< feature-state feature_gate_name="DeploymentPodReplacementPolicy" >}}
+
+{{< note >}}
+This feature is disabled by default. You can enable it by setting the `DeploymentPodReplacementPolicy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
+and on the [Kube Controller Manager](/docs/reference/command-line-tools-reference/kube-controller-manager/)
+{{< /note >}}
+
+Deployment Pods can become terminating due to deletion, usually during the rollout or scaling of a new
+deployment. Some pods may take a long time to terminate and may consume additional resources during this
+time. As a result, the total number of all pods can temporarily exceed `.spec.replicas`
+(or `.spec.replicas + .spec.strategy.rollingUpdate.maxSurge` for RollingUpdate Deployments).
+A `.spec.podReplacementPolicy` field can be used to control when a Deployment should create new pods.
```

Comment:
```suggestion
Providing this feature is enabled in your cluster, you can specify a `.spec.podReplacementPolicy`
to control when a Deployment should create new pods. The field affects the behavior of the
Deployment during scaling operations and when rolling out updates.
```

Reply:
- Do we have to mention the enablement the 2nd time? I would think it is implied for the rest of the section.
- Same question about the last part. Should we mention it again? We also have it at the end.